src/client/components/forms/deletion.js

```diff
@@ -19,10 +19,10 @@
 import * as bootstrap from 'react-bootstrap';
 
 import CustomInput from '../../input';
-import FontAwesome from 'react-fontawesome';
 import LoadingSpinner from '../loading-spinner';
 import PropTypes from 'prop-types';
 import React from 'react';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
```

Comment:
This import needs to be in place of old import , on the same line 22 , so that all the imports are sorted alphabetically.